### PR TITLE
[ja] brash up expressions of explanation of controllers

### PR DESCRIPTION
**Pull request by iterable-company**
Current expressions are very simplified, and that makes it harder to understand the intent of the original expression.
More details are added to approximate the intent of the original text.

### CODE REVIEW

File: content/ja/docs/concepts/overview/components.md [hunk 1]
```diff
@@ -58,11 +58,11 @@ Kubernetesをオンプレミスあるいは個人のPC内での学習環境で
 
 kube-controller-managerと同様に、cloud-controller-managerは複数の論理的に独立したコントロールループをシングルバイナリにまとめ、一つのプロセスとして動作します。パフォーマンスを向上させるあるいは障害に耐えるために水平方向にスケールする(一つ以上のコピーを動かす)ことができます。
 
-次のコントローラーは、クラウドプロバイダーへの依存関係を持つことがあります。
+次の各コントローラは、それぞれ以下に示すような目的のためにクラウドプロバイダへの依存関係を持つことができるようになっています。
```
Comment (nasa9084) [inline content/ja/docs/concepts/overview/components.md:61]:
```suggestion
次の各コントローラーは、それぞれ以下に示すような目的のためにクラウドプロバイダーへの依存関係を持つことができるようになっています。
```

File: content/ja/docs/concepts/overview/components.md [hunk 2]
```diff
@@ -58,11 +58,11 @@ Kubernetesをオンプレミスあるいは個人のPC内での学習環境で
...
 
-  * Nodeコントローラー：ノードが応答を停止した後、クラウドで削除されたかどうかを判断するため、クラウドプロバイダーをチェックします。
-  * Routeコントローラー：基盤であるクラウドインフラでルーティングを設定します。
-  * Serviceコントローラー：クラウドプロバイダーのロードバランサーの作成、更新、削除を行います。
+  * Nodeコントローラ: ノードが応答を停止した後、クラウドで当該ノードが削除されたかどうかを判断するため
```
Comment (nasa9084) [inline content/ja/docs/concepts/overview/components.md:63]:
```suggestion
  * Nodeコントローラー: ノードが応答を停止した後、クラウドで当該ノードが削除されたかどうかを判断するため
```

File: content/ja/docs/concepts/overview/components.md [hunk 3]
```diff
@@ -58,11 +58,11 @@ Kubernetesをオンプレミスあるいは個人のPC内での学習環境で
...
+  * Route コントローラ: 基盤となるクラウドインフラのルートを設定するため
```
Comment (nasa9084) [inline content/ja/docs/concepts/overview/components.md:64]:
```suggestion
  * Route コントローラー: 基盤となるクラウドインフラのルートを設定するため
```

File: content/ja/docs/concepts/overview/components.md [hunk 4]
```diff
@@ -58,11 +58,11 @@ Kubernetesをオンプレミスあるいは個人のPC内での学習環境で
...
+  * Service コントローラ: クラウドプロバイダーのロードバランサの作成、更新、削除を行うため
```
Comment (nasa9084) [inline content/ja/docs/concepts/overview/components.md:65]:
```suggestion
  * Service コントローラー: クラウドプロバイダーのロードバランサーの作成、更新、削除を行うため
```

**Review by nasa9084 — COMMENTED** [review]
/assign

File: content/ja/docs/concepts/overview/components.md (hunk 1 above)
Comment (iterable-company) [reply content/ja/docs/concepts/overview/components.md]: Sorry, I follow the conventions of the translation about 'er'.

File: content/ja/docs/concepts/overview/components.md (hunk 2 above)
Comment (iterable-company) [reply content/ja/docs/concepts/overview/components.md]: Same as mentioned above.

File: content/ja/docs/concepts/overview/components.md (hunk 3 above)
Comment (iterable-company) [reply content/ja/docs/concepts/overview/components.md]: Same as mentioned above.

File: content/ja/docs/concepts/overview/components.md (hunk 4 above)
Comment (iterable-company) [reply content/ja/docs/concepts/overview/components.md]: Same as mentioned above.

**Review by nasa9084 — COMMENTED** [review]
/approve

**Pull request merged**